apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -31,6 +32,12 @@ import { MANAGER_TABS } from "../../const/manager";
 import ExternalLink from "../ExternalLink";
 import { track } from "../../analytics";
 import TermsFooter, { TermsProviders } from "../TermsFooter";
+import Illustration from "../../images/illustration/Illustration";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const notOnboardedDarkImg = require("../../images/illustration/Dark/_010.png");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const notOnboardedLightImg = require("../../images/illustration/Light/_010.png");
```

Comment:
Why is this a require and not an import like elsewhere in the app? 🤔 
`import lightPlaceholderImage from "../../images/illustration/Light/_000_PLACEHOLDER.png";`

Reply:
That's a very good question. I can't remember anymore, I guess I inspired myself in some other code which had something similar to this. In any case, I've fixed it and tested it, all working well.